### ISSUE TITLE
Sorting Downloads by date created (descending) makes the file view disappear

### ISSUE THREAD

**Issue by RUvlad1** [spacedriveapp/spacedrive#1540]
### Describe the bug

Exactly what the title says. When I go to my Downloads folder and sort by date created, the file view disappears.

Here's a video of the bug in action
https://github.com/spacedriveapp/spacedrive/assets/60317461/a12cb669-97f6-4bc8-8996-e7c2f881e30e

### Reproduction

1. Open the Downloads folder
2. Sort by Date Created (descending)

### Expected behavior

The files would sort correctly

### Platform and versions

```Shell
Windows 10 LTSC 21H2 (19044.3570)

Spacedrive v0.1.0 - fdc3c30
```

### Additional context

I can fix this by going back a page twice and then open Downloads again, but that resets the sort, which isn't really great cause I'm more used to how explorer does it in Windows.

This is also probably because my Downloads folder is a little too big... well, about 70 GB in size

**Comment by utkubakir** [comment]
Size shouldn't really matter for sorting it as long as it's indexed. We will look into this!

**Comment by nprasad2077** [comment]
Yeah strange, i also have a similar issue on the latest version. Absolutely nothing is sorting properly in any of the folders. All database/indexing operations seem to be complete. Nothing pending. 

My download folder is about 10gb right now and my Documents are only 1GB. 

Bug is on Windows 11 right now, on a pretty clean system. Hopefully something resolves!

**Comment by HeavenVolkoff** [comment]
Is this still happening with the latest version?

**Comment by jamiepine** [comment]
Closing due to v2 rewrite. If you encounter this issue in v2, please open a new issue.
